tests/spec/core/lint-hrefs-ids-spec.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+describe("Core — Lint href's matching id's", function() {
+  afterAll(function(done) {
+    flushIframes();
+    done();
+  });
+  it("should warn about href pointing nowhere", function(done) {
```

Comment:
nit: should be "warns if can't match a fragment identifier to an id in the document"